test/e2e/testing-manifests/glbc/rc.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: glbc
+  labels:
+    app: glbc
+    version: v1
+spec:
+  # There should never be more than 1 controller alive simultaneously.
+  replicas: 1
+  selector:
+    app: glbc
+    version: v1
+  template:
+    metadata:
+      labels:
+        app: glbc
+        version: v1
+    spec:
+      # It only needs this long if started with --delete-all-on-quit.
+      terminationGracePeriodSeconds: 600
+      containers:
+      - image: bprashanth/glbc:0.0
```

Comment:
need to upload to gcr.io, this is the gce controller being tested https://github.com/kubernetes/contrib/tree/master/Ingress/controllers/gce